locales/it/index.yml

```diff
@@ -3370,6 +3370,8 @@ features:
           body: "Perderai l’Identità Digitale e tornerai allo stato iniziale disattivando IT Wallet. Potrai riattivarlo in un secondo momento."
           confirm: "Conferma reset"
           cancel: "Annulla"
+      buttons:
+        addCredential: "Aggiunti documento"
```

Comment:
Should it be "Aggiungi tessera"?

Reply:
documento/i now should be tessera as in Figma

Reply:
> Note: currently the FIGMA is not updated with the latest copy content, thus it still uses the word "Tessera" instead of "Documento".

Reply:
fix 
```suggestion
        addCredential: "Aggiungi documento"
```